.github/workflows/ci.yml

```diff
@@ -497,7 +497,7 @@ jobs:
       - basic
       - dependency
       - license
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

Comment:
minor comment but can we define & initialise a variable in env section and use it throughout the file

Reply:
This will be improved in #7497. ;)